terraform_compliance/steps/terrain.py

```diff
@@ -124,13 +126,35 @@ def wait_for_user_input(step):
             embed()
         elif cmd == 's':
             print(json.dumps(step.context.stash, indent=4))
+        elif cmd == 'l':
```

Comment:
hmm does this change the context of the main run, while we run some steps with debugger ? 🤔

Reply:
Yes. Adding the following line to the documentation. "Injecting a step via `l` to the execution will modify the context, stash, and whether the scenario is skipped or failed, depending on the behavior of the step."

Reply:
hmm I think for the sake of `debugging` we need to find a way to isolate this.